plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -9,6 +9,8 @@ deploy.login.dialog.message=Browser opened to authorize deployment.
 deploy.manual.link=To manually promote a version use the <a href="{0}">Google Cloud Console.</a>
 deploy.preferences.dialog.title=Deployment configuration
 deploy.preferences.dialog.title.withProject=Deployment configuration for {0}
+deploy.preferences.dialog.label.selectCredential=Credential to use:
+deploy.preferences.dialog.accountSelector.login=<Add a new account...>
```

Comment:
I occasionally saw the angle brackets for a special "value" in a combo box in order to distinguish it from other actual values. However, I'll remove them as I don't have any objection doing so.